minecraft_servers/spieleoase/manifest.json

```
@@ -0,0 +1,16 @@
+{
+  "server_name": "spieleoase",
+  "nice_name": "SpieleOase",
+  "direct_ip": "play.spieleoase.net",
+  "server_wildcards": [
+    "%.spieleoase.net",
+    "%.spieleoase.eu"
+  ],
+  "social": {
+    "web": "https://spieleoase.net/",
+    "twitter": "SpieleOaseNET",
+	"youtube": "https://www.youtube.com/channel/UCdYenysz26oUGg83H6MxHAQ",
```

Please do not use tabs, use spaces instead.
```suggestion
    "youtube": "https://www.youtube.com/channel/UCdYenysz26oUGg83H6MxHAQ",
    "instagram": "https://www.instagram.com/spieleoasenet",
```